packages/xdl/src/detach/IosNSBundle.js

```diff
@@ -517,8 +513,7 @@ async function configureAsync(context: StandaloneContext) {
         await AssetBundle.bundleAsync(
           context,
           context.data.manifest.bundledAssets,
-          supportingDirectory,
-          context.data.manifest.sdkVersion === '24.0.0'
```

Comment:
Also removed this argument because the support for SDK 24 has been dropped a long time ago. See https://github.com/expo/expo-cli/pull/955/files#diff-0c6590ff0c930eda50db2e4a5ecf83b1L21